liqd_product/config/settings/base.py

```diff
@@ -270,6 +292,8 @@
     }
 }
 
+# Wagtail
+WAGTAIL_SITE_NAME = "liquid product"
```

Comment:
should be "Beteiligung.in" now

Reply:
I was not sure here - 'beteiligen' or 'beteiligung'